docs/cudf/source/basics/nested.rst

```diff
@@ -0,0 +1,35 @@
+Nested
```

Comment:
The title does not have to match the file name -- let's make the title `Struct and List datatypes`.